console/src/components/Services/Data/TableBrowseRows/EditItem.js

```diff
@@ -63,7 +67,6 @@ class EditItem extends Component {
 
     const elements = columns.map((col, i) => {
       const colName = col.column_name;
-      const colType = col.data_type;
       const hasDefault = col.column_default && col.column_default.trim() !== '';
```

Comment:
`hasDefault` could be a prop to `TypedInput`. Because you are redoing this inside it too.
Right?